integration-tests/smoke/automation_test.go

```diff
@@ -149,13 +150,31 @@ func TestAutomatedBasic(t *testing.T) {
 		}
 	}, "5m", "1s").Should(gomega.Succeed()) // ~1m for cluster setup, ~2m for performing each upkeep 5 times, ~2m buffer
 
+	if nodeUpgrade {
+		actions.UpgradeChainlinkNodeVersions(testEnv, upgradeImage, upgradeVersion,
+			chainlinkNodes[0], chainlinkNodes[1], chainlinkNodes[2], chainlinkNodes[3], chainlinkNodes[4])
```

Comment:
Could the chainlinkNodes args be simplified with something like `chainlinkNodes[0:4]...`